docs/tutorials/other-resources/references.md

```diff
@@ -0,0 +1,61 @@
+# Additional References
+Below is a list of additional resources from the community.
+
+# Further Reading
```

Comment:
There are two `#`. We need to remove this line or remove the first line.

Reply:
removed this line :)